lmdeploy/turbomind/deploy/source_model/internvl.py

```diff
@@ -94,6 +94,11 @@ class InternVLAwqReader(LlamaAwqReader):
     norm_weight_key = 'language_model.model.norm.weight'
     output_weight_key = 'language_model.lm_head.weight'
 
+    def __init__(self, new_params: dict, unused_params: dict, last_bin: bool,
+                 model_cfg: dict):
+        model_cfg = model_cfg.get('llm_config')
```

Comment:
Hi @AllentDan I compared https://huggingface.co/OpenGVLab/InternVL2-40B/blob/main/config.json and https://huggingface.co/OpenGVLab/InternVL2-26B/blob/main/config.json. Both of their configurations have `llm_config`. Why was the InternVL2 26B model working fine before this fix?

Reply:
`architectures` in `llm_config` differs.

Reply:
Yes, these two models use different architectures.